Jellyfin.Server/Extensions/ApiApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Builder;
+
+namespace Jellyfin.Server.Extensions
+{
+    /// <summary>
+    /// Extensions for adding API specific functionality to the application pipeline.
+    /// </summary>
+    public static class ApiApplicationBuilderExtensions
+    {
+        /// <summary>
+        /// Adds swagger and swagger UI to the application pipeline.
+        /// </summary>
+        /// <param name="applicationBuilder">The application builder.</param>
+        /// <returns>The updated application builder.</returns>
+        public static IApplicationBuilder UseJellyfinApiSwagger(this IApplicationBuilder applicationBuilder)
+        {
+            applicationBuilder.UseSwagger();
+
+            // Enable middleware to serve swagger-ui (HTML, JS, CSS, etc.),
+            // specifying the Swagger JSON endpoint.
+            return applicationBuilder.UseSwaggerUI(c =>
+            {
+                c.SwaggerEndpoint("/swagger/v1/swagger.json", "Jellyfin API V1");
```

Comment:
won't this break if user has baseURL set?

Reply:
Depends on what you mean. Swagger will be available at /swagger